aws/network-primitives/scripts/create_bootstrap_node_evm_compose_file.sh

```diff
@@ -155,22 +151,19 @@ if [ "${enable_domains}" == "true" ]; then
     {
     # core domain
       echo '      "--",'
-      echo '      "--chain=${NETWORK_NAME}",'
+      echo '      "--domain-id=${DOMAIN_ID}",'
       echo '      "--node-key", "${NODE_KEY}",'
-    #  echo '      "--enable-subspace-block-relay",'
       echo '      "--state-pruning", "archive",'
       echo '      "--blocks-pruning", "archive",'
-      echo '      "--listen-addr", "/ip4/0.0.0.0/tcp/${OPERATOR_PORT}",'
-      echo '      "--domain-id=${DOMAIN_ID}",'
+      echo '      "--listen-on", "/ip4/0.0.0.0/tcp/${OPERATOR_PORT}",'
       echo '      "--base-path", "/var/subspace/core_${DOMAIN_LABEL}_domain",'
```

Comment:
If this is a container, why do you have this custom path rather than `/var/subspace`?